.storybook/stories/KvCarousel.stories.js

```diff
@@ -0,0 +1,395 @@
+import Vue from 'vue'
+import Vue2TouchEvents from 'vue2-touch-events';
+
+import KvCarousel from '@/components/Kv/KvCarousel';
+import KvCarouselSlide from '@/components/Kv/KvCarouselSlide';
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+
+Vue.use(Vue2TouchEvents);
+
+const defaultCarouselSlides = `
+	<template slot="default" slot-scope="props">
+		<kv-carousel-slide :transition-name="props.transitionName">
+			<img src="https://placekitten.com/300/290" style="width: 100%;">
```

Comment:
Let's change placekitten to https://via.placeholder.com/{width}x{height}/{background-color-hexcode} (I know, I'm no fun)

Reply:
😿